### DISCUSSION TITLE
I can't login via my email.

### DISCUSSION THREAD

**Discussion by lesvere** [vercel/community#3408] [Help] [answered]
### Steps to Reproduce

looking forward your reply.

**Comment by lesvere** [comment]
Server response "This user account is blocked."
WHY?

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.

**Comment by lesvere** [comment]
OK thx
I’m already sent the mail

Sent from my iPhone

On 28 Jul 2023, at 9:16 PM, Amy ***@***.***> wrote:

﻿

Please email ***@***.******@***.***> and include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
